catalog/spatial/geocoding/spatial-geocoding-localsearch/src/main/java/org/codice/ddf/spatial/geocoder/geonames/GeoNamesLocalIndex.java

```diff
@@ -60,7 +60,8 @@ public GeoResult getLocation(final String location) {
                         population);
             }
         } catch (GeoEntryQueryException e) {
-            LOGGER.error("Error querying the local GeoNames index", e);
+            LOGGER.error(
+                    "Error querying the local GeoNames index: load a Geonames file into the offline gazetteer");
```

Comment:
Will this error message also be thrown when the query cannot be parsed?  https://github.com/peterhuffer/ddf/blob/DDF-1922/catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/java/org/codice/ddf/spatial/geocoding/query/GeoNamesQueryLuceneIndex.java#L140